pkg/controller/plan/kubevirt.go

```diff
@@ -1744,6 +1745,29 @@ func (r *KubeVirt) guestConversionPod(vm *plan.VMStatus, vmVolumes []cnv.Volume,
 					Type: core.SeccompProfileTypeRuntimeDefault,
 				},
 			},
+			Affinity: &core.Affinity{
+				PodAntiAffinity: &core.PodAntiAffinity{
+					PreferredDuringSchedulingIgnoredDuringExecution: []core.WeightedPodAffinityTerm{
+						{
+							Weight: 100,
+							PodAffinityTerm: core.PodAffinityTerm{
+								TopologyKey: "kubernetes.io/hostname",
+								LabelSelector: &metav1.LabelSelector{
+									MatchExpressions: []metav1.LabelSelectorRequirement{
+										{
+											Key: kApp,
```

Comment:
kApp - sounds starange, is this the curect key ?

Reply:
maybe `app` ?

Reply:
The `kApp` is a variable which is set to the pod in `conversionLabels` as a key of label

Reply:
https://github.com/kubev2v/forklift/blob/3ad243d6caa4fac32f6ea18b1158241794701bb8/pkg/controller/plan/kubevirt.go#L2235